### PR TITLE
Proposing Harbor for donation to CNCF

### PR DESCRIPTION
**Pull request by clouderati**
Hi all! Here's a proposal for Harbor to join the CNCF. Please review and provide feedback.

Many thanks!

### CODE REVIEW

**Comment by caniszczyk** [comment]
Thanks @clouderati RFC: @cncf/toc (we already discussed this on the call and have two TOC sponsors for the sandbox). I'll leave this open for about a week for community feedback

File: proposals/harbor.adoc [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+ * *Infrastructure Required:* Infrastructure for CI / CD
+ * *Website:* https://vmware.github.io/harbor/. Will be moved to https://goharbor.io.
+ * *Release Methodology and Mechanics:* We currently do feature releases for major updates 3-4 times per year (with minor releases) when needed. Before releasing we tag one or more RC releases for community testing. Commits to the project are analyzed and we require that changes do not decrease overall test coverage to the project.
+
+== Social Media Accounts:
+
+ * *Twitter:* https://twitter.com/project_harbor
+ * *Users Google Groups:* harbor-users@googlegroups.com
+ * *Developer Google Groups:* harbor-dev@googlegroups.com
+ * *Slack:* #harbor on https://code.vmware.com/join/
```
Comment (caniszczyk) [inline proposals/harbor.adoc]: this will have to move to the CNCF slack or a neutral slack

Comment (clouderati) [reply proposals/harbor.adoc]: Yes, absolutely. We're in the process of setting up a new Slack workspace @caniszczyk.

File: proposals/harbor.adoc [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+ * Governance – General access to staff to provide advice, and help optimize and document our governance process
+ * Infrastructure for CI / CD
+ * Integration with CNCF devstat
+ * A vendor-neutral home for Harbor
+
+
+== Appendices
+
+=== Architecture
+Harbor is cleanly architected and includes both third-party components – notably Clair, Notary and Nginx – and various Harbor-specific components. Harbor leverages Kubernetes to manage the runtimes of the various components.
```
[the remark this answers is not in the document]
Comment (clouderati) [reply proposals/harbor.adoc]: You're absolutely right @justincormack. Will add the Docker registry component to the list.

**Pull request merged**